### ISSUE TITLE
init with --template typescript creates template folder

### ISSUE THREAD

**Issue by alexakasanjeev** [react-native-community/cli#764]
## Ask your Question
I'm trying to create a new project with typescript, it is mentioned to used 
```
react-native init project --template typescript
```

but when I'm doing that, there is template folder inside project, which again contains a react native project that contains typescript related code. Please check the repo, the entire files are created using above command, and when you will open template folder, you will see it is again one more project

So basically there is a react-native project inside react-native project
here is the repo https://github.com/alexakasanjeev/wallet

**Comment by thymikee** [comment]
Please follow the instructions from the template itself 🙂: https://github.com/react-native-community/react-native-template-typescript/#arrow_forward-usage

**Comment by alexakasanjeev** [comment]
Thank you for the help :smile:
